### ISSUE TITLE
Panic for "integer divide by zero" for empty file

### ISSUE THREAD

**Issue by oneumyvakin** [glaslos/ssdeep#1]
**STEPS**
1. Create empty file empty.txt.
2. Try to execute following code:
```
deepSig := ssdeep.NewSSDEEP()
sigStr := deepSig.Fuzzy("empty.txt")
```

**ACTUAL RESULT**
```
panic: runtime error: integer divide by zero
goroutine 1 [running]:
github.com/glaslos/ssdeep.(*SSDEEP).Fuzzy(0xc042035e70, 0x4d68ce, 0xb, 0x0, 0x0)
	/GoPath/src/github.com/glaslos/ssdeep/ssdeep.go:137 +0x3f0
```

**Comment by glaslos** [comment]
Hey, thanks for reporting those issues. Do you have time to look into them?

**Comment by glaslos** [comment]
Fixed in 1e2b11fb27cbbc18aabcff4f0953058e5dea787f
